src/strat_engine/serialize.rs.in

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is for our on-disk metadata format, which is currently defined as JSON. I'll rename the file and add an explanatory comment.